This is an upstream patch for https://github.com/codership/galera/pull/617
"remove duplicate va_end call" that originates from BZ #2051849 covscan
results. The upstream has not addressed the issue for over a year so this
is the reason to cover it as separate patch here.

--- galera-25.3.37/galerautils/src/gu_dbug.c	2022-05-31 17:05:23.000000000 +0200
+++ ../../galera-25.3.37/galerautils/src/gu_dbug.c	2023-07-10 17:09:58.969599380 +0200
@@ -1040,7 +1040,6 @@ _gu_db_doprnt_(const char *format, ...)
 	}
 	(void) fprintf(_gu_db_fp_, "%s: ", state->u_keyword);
 	(void) vfprintf(_gu_db_fp_, format, args);
-	va_end(args);
 	(void) fputc('\n', _gu_db_fp_);
 	dbug_flush(state);
 	errno = save_errno;

